config/config.go

```diff
@@ -49,10 +54,10 @@ type (
 
 	// SystemFee fees related to system.
 	SystemFee struct {
-		EnrollmentTransaction int64 `yaml:"EnrollmentTransaction"`
-		IssueTransaction      int64 `yaml:"IssueTransaction"`
-		PublishTransaction    int64 `yaml:"PublishTransaction"`
-		RegisterTransaction   int64 `yaml:"RegisterTransaction"`
+		EnrollmentTransaction int `yaml:"EnrollmentTransaction"`
+		IssueTransaction      int `yaml:"IssueTransaction"`
+		PublishTransaction    int `yaml:"PublishTransaction"`
+		RegisterTransaction   int `yaml:"RegisterTransaction"`
```

Comment:
it must be int64

Reply:
or.. mb float64

Reply:
in the C# implementation is [int64](https://github.com/neo-project/neo/blob/master/neo/Fixed8.cs#L29)

```csharp
public Fixed8(long data)
        {
            this.value = data;
        }
```
Why do you suggest `float64`?

Shall I change everything so that the `NewFixed8` method accept int64?
```go
// NewFixed8 return a new Fixed8 type multiplied by decimals.
func NewFixed8(val int64) Fixed8 {
	return Fixed8(decimals * val)
}
```

Reply:
@dauTT yep, `long` / `int64` for new.. but...  

```csharp
public Fixed8(long data)
        {
            this.value = data;
        }
```

and 

```csharp
        public static Fixed8 FromDecimal(decimal value)
        {
            value *= D;
            if (value < long.MinValue || value > long.MaxValue)
                throw new OverflowException();
            return new Fixed8
            {
                value = (long)value
            };
        }
```

Reply:
```
private const long D = 100_000_000;
```

Reply:
okay, I think I understood what you mean.